packages/core/src/browser/json-schema-store.ts

```diff
@@ -38,6 +38,9 @@ export class JsonSchemaStore {
     protected readonly onSchemasChangedEmitter = new Emitter<void>();
     readonly onSchemasChanged = this.onSchemasChangedEmitter.event;
 
+    protected readonly onDidChangeSchemaEmitter = new Emitter<string>();
```

Comment:
let's keep `URI` here, clients can do `toString` if it is required or check other properties